### ISSUE TITLE
Automated PRs?

### ISSUE THREAD

**Issue by svandragt** [kylemanna/docker-syncthing-relay#3]
Would it be a good idea to automate the SyncThing Relay Server version once a week or something?

It should be possible to write a script that:
* Retrieves the RSS feed at https://build.syncthing.net/job/strelaysrv/rssAll
* Extract the build number from the entry title and somehow check the build was successful (?)
* Scrape the Jenkins build page for the full version number
* Create a PR with the updated Dockerfile

This docker container is the only auto updating SyncThing Relay Server solution I'm aware of, and that worries me in case there ever are security fixes that need to be rolled out.

**Comment by kylemanna** [comment]
Might be better to subscribe to source code repo updates (is there an easy way to do this with a Github API?) and just build it in the Docker image rather then download it?  As it stands now, the [`strelaysrv` cmd](https://github.com/syncthing/syncthing/commits/master/cmd/strelaysrv) hasn't been updated in over a month.

Perhaps the most logical and simplest thing to do is to update the Docker image when there is a [release](https://github.com/syncthing/syncthing/releases)?

Thoughts?

**Comment by svandragt** [comment]
Ok I did some digging. Initially, it was not clear to me if the strelaysrv releases are linked to syncthing releases, or that the code happens to be in the repo but the build system is the source of strelaysrv releases. I was assuming the latter based on https://forum.syncthing.net/t/starting-relayserver/8493/7 however you're implying the former. It turns out you're right:

I explored the build system and it appears the builds are building off the syncthing repo (see https://build.syncthing.net/job/strelaysrv/614/git/) so you're right that the code has not been updated in a month, so the build system is building new builds only because syncthing itself changed rather than the relay server component.

With that in mind it makes sense to track the github releases as you proposed under the assumption that important security fixes in relay server would result in a new syncthing release.

I don't know enough about docker to answer your question on whether it's possible to 'track' github releases. I found https://docs.docker.com/docker-hub/github/ but I think it assumes the person building the docker images is the one controlling github? If that is the way to go that means forking the repo and pulling upstream changes automatically on release.

Hope that helps
